packages/client/components/ReflectionCard/ReflectionCardRoot.tsx

```diff
@@ -11,8 +10,8 @@ const ReflectionCardRoot = styled('div')({
   display: 'block',
   maxWidth: '100%',
   position: 'relative',
-  transition: `box-shadow 2000ms ${DECELERATE}`,
-  width: ElementWidth.REFLECTION_CARD
+  transition: `all 100ms ${BezierCurve.DECELERATE}`,
```

Comment:
box-shadow should still animate in at 2s. a transition can support different speeds & curves for different props